app/components/collections/header_component.html.erb

```diff
@@ -1,5 +1,6 @@
 <header class="title">
   <span class="header-text"><%= name %></span>
+  <%= render Collections::EditLinkComponent.new(collection: collection, anchor: '#', label: "Edit #{name}") %>
```

Comment:
Won't this result in a link like: `http://example.com/path##` with two octothorpes?

Reply:
Yes, good catch, I missed this one - I fixed it now.